src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -1293,18 +1303,18 @@ private Bid toBid(BidInfo bidInfo, ExtRequestTargeting targeting, BidRequest bid
         final ExtResponseCache cache = bids != null || vastXml != null ? ExtResponseCache.of(bids, vastXml) : null;
 
         final ObjectNode originalBidExt = bid.getExt();
-        final ExtBidPrebid extBidPrebid = getExtPrebid(originalBidExt);
-        final ExtBidPrebid.ExtBidPrebidBuilder extBidPrebidBuilder = extBidPrebid != null
-                ? extBidPrebid.toBuilder()
-                : ExtBidPrebid.builder();
         final Boolean dealsTierSatisfied = bidInfo.getSatisfiedPriority();
 
-        final ExtBidPrebid updatedExtBidPrebid = extBidPrebidBuilder
-                .targeting(MapUtils.isEmpty(targetingKeywords) ? null : targetingKeywords)
-                .targetBidderCode(targetingInfo.isAddTargetBidderCode() ? bidderCode : null)
-                .dealtiersatisfied(dealsTierSatisfied)
-                .cache(cache)
-                .build();
+        final ExtBidPrebid updatedExtBidPrebid =
+                getExtPrebid(originalBidExt, ExtBidPrebid.class)
+                        .map(ExtBidPrebid::toBuilder)
+                        .orElseGet(ExtBidPrebid::builder)
+                        .targeting(MapUtils.isEmpty(targetingKeywords) ? null : targetingKeywords)
```

Comment:
Nitpick. 1312 1313, pls make null position same for both lines, would appreciate if you put it on second position